CHANGES.md

```diff
@@ -75,6 +75,8 @@ Tactics
   foo : database`). When the database name is omitted, the hint is added to the
   core database (as previously), but a deprecation warning is emitted.
 
+- Ltac backtraces can be turned on using the "Ltac Backtrace" option.
```

Comment:
Ah, this is a flag (meaning it is on or off).  Options take values, such as a string or a number.  Certainly a detail, though.

Reply:
```suggestion
- Ltac backtraces can be turned on using the `Ltac Backtrace` flag.
```